contracts/seed/Seed.sol

```diff
@@ -167,12 +167,12 @@ contract Seed {
         uint lockedSeedFee = (alreadyLockedSeedTokens.mul(uint(PPM))).mul(fee).div(PPM100);
 
         // We are that overall supply of fundingToken that will be in the end of this function execution
```

Comment:
pls fix or delete the comment, it's not clear what it is saying

Reply:
Fixed this comment with PR #117